DECISION-MAKING.md

```diff
@@ -0,0 +1,86 @@
+# Decision-making rules
+
+## Introduction
+
+The Brain Imaging Data Structure (BIDS) community set out the following 
+decision-making rules with the intention to:
+
+- Strive for consensus.
+- Promote open discussions.
+- Minimize the administrative burden.
+- Provide a path for when consensus cannot be made.
+- Grow the community.
+- Maximize the [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of the 
+  project.
+
+The rules outlined below are inspired by the [lazy consensus system used in the Apache Foundation](https://www.apache.org/foundation/voting.html) 
+and heavily depends on [GitHub Pull Request Review system](https://help.github.com/articles/about-pull-requests/).
+
+## Definitions
+
+**Repository** - [https://github.com/bids-standard/bids-specification](https://github.com/bids-standard/bids-specification)
+
+**Contributor** - a person listed in the Appendix I: Contributors. The 
+community decides on the content of this file using the same process as any 
+other change to the Repository (see below) allowing the meaning of "Contributor"
+to evolve independently of the Decision-making rules.
+
+## Rules
+
+1. Every modification of the specification (including a correction of a typo, 
+   adding a new Contributor, an extension adding support for a new data type, or
+   others) or proposal to release a new version needs to be done via a Pull 
+   Request (PR) to the Repository.
+1. Anyone can open a PR (this action is not limited to Contributors).
+1. PRs adding new Contributors must also add their GitHub names to the 
+   [CODEOWNERS](CODEOWNERS) file.
+1. A PR is eligible to be merged if and only if these conditions are met:
+   1. The last commit is at least 5 working days old to allow the community to 
+      evaluate it.
+   1. The PR features at least one [Review that Approves](https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews) 
+      the PR from a Contributor that is not an author of the PR. The review 
+      needs to be made after the last commit in the PR (equivalent to 
+      [Stale review dismissal](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/)
+      option on GitHub).
+   1. Does not feature any [Reviews that Request changes](https://help.github.com/articles/about-required-reviews-for-pull-requests/).
+   1. Does not feature "WIP" in the title (Work in Progress).
+   1. Passes all automated tests.
+1. Any Contributor can Review a PR and Request changes. If a Contributor 
```

Comment:
Are `Contributor`, `Review` and `Request` all capitalised for a reason? Because they have particular meaning?

Reply:
Yes, but I am not married to this particular capitalization. Please propose a different capitalization if you find it more clear.

Reply:
I like the idea of capitalising words that are in the definitions section at the top.... and I also like the idea of defining `Review` and `Request` but I'm not super sure I could do it at the moment...

(This isn't super helpful... maybe it could be an issue for future consideration?)